ai/services.py

```
@@ -0,0 +1,97 @@
+import os
+import google.generativeai as genai
+import openai
+from typing import Dict, Any, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)